packages/protocol/contracts/common/Accounts.sol

```diff
@@ -275,7 +275,8 @@ contract Accounts is IAccounts, Ownable, ReentrancyGuard, Initializable, UsingRe
   }
 
   /**
-   * @notice Removes the currently authorized vote signer for the account
+   * @notice Removes the currently authorized vote signer for the account.
+   * Note that the signers cannot be reauthorized after they have been removed.
```

Comment:
is this by design?  we don't want signers to be able to reauthorize?

Reply:
According to @asaj's comment in the issue, it's by design.